doc/releases/changelog-0.40.0.md

```diff
@@ -437,61 +437,148 @@ such as `shots`, `rng` and `prng_key`.
+  Using new Resource versions of existing operations and `get_resources`, we can estimate
+  resources quickly:
 
-<h4>Experimental functionality for handling dynamical Lie algebras (DLAs)</h4>
+  ```python
+  import pennylane.labs.resource_estimation as re
+  
+  def my_circuit():
+      for w in range(2):
+          re.ResourceHadamard(w)
+      re.ResourceCNOT([0, 1])
+      re.ResourceRX(1.23, 0)
+      re.ResourceRY(-4.56, 1)
+      re.ResourceQFT(wires=[0, 1, 2])
+      return qml.expval(re.ResourceHadamard(2))
+  ```
+  ```pycon
+  >>> res = re.get_resources(my_circuit)()
+  >>> print(res)
+  wires: 3
+  gates: 202
+  gate_types:
+  {'Hadamard': 5, 'CNOT': 10, 'T': 187}
+  ```
+  We can also set custom gate sets for decompositions:
+  ```pycon
+  >>> gate_set={"Hadamard","CNOT","RZ", "RX", "RY", "SWAP"}
+  >>> res = re.get_resources(my_circuit, gate_set=gate_set)()
+  >>> print(res)
+  wires: 3
+  gates: 24
+  gate_types:
+  {'Hadamard': 5, 'CNOT': 7, 'RX': 1, 'RY': 1, 'SWAP': 1, 'RZ': 9}
+  ```
+  Alternatively, it is possible to manually substitute associated resources:
+  ```pycon
+  >>> new_resources = re.substitute(res, "SWAP", re.Resources(2, 3, {"CNOT":3}))
+  >>> print(new_resources)
+  {'Hadamard': 5, 'CNOT': 10, 'RX': 1, 'RY': 1, 'RZ': 9}
+  ```
 
-* Added a dense implementation of computing the Lie closure in a new function
-  `lie_closure_dense` in `pennylane.labs.dla`.
-  [(#6371)](https://github.com/PennyLaneAI/pennylane/pull/6371)
-  [(#6695)](https://github.com/PennyLaneAI/pennylane/pull/6695)
 
-* Added a dense implementation of computing the structure constants in a new function
-  `structure_constants_dense` in `pennylane.labs.dla`.
-  [(#6376)](https://github.com/PennyLaneAI/pennylane/pull/6376)
+<h4>Experimental functionality for handling dynamical Lie algebras (DLAs)</h4>
 
-* Added utility functions for handling dense matrices and advanced functionality in the Lie theory context.
-  [(#6563)](https://github.com/PennyLaneAI/pennylane/pull/6563)
+* Use the `pennylane.labs.dla` module to perform the
+  [KAK decomposition](https://pennylane.ai/qml/demos/tutorial_kak_decomposition):
+  * `cartan_decomp`: obtain a **Cartan decomposition** of an input **Lie algebra** via an **involution**
+  [(#6392)](https://github.com/PennyLaneAI/pennylane/pull/6392)
+  * We provide a variety of **involutions** like `concurrence_involution`, `even_odd_involution` and canonical Cartan involutions
   [(#6392)](https://github.com/PennyLaneAI/pennylane/pull/6392)
   [(#6396)](https://github.com/PennyLaneAI/pennylane/pull/6396)
+  * `cartan_subalgebra`: compute a horizontal **Cartan subalgebra**
+  [(#6403)](https://github.com/PennyLaneAI/pennylane/pull/6403)
+  * `variational_kak_adj`: compute a [variational KAK decomposition](https://pennylane.ai/qml/demos/tutorial_fixed_depth_hamiltonian_simulation_via_cartan_decomposition) of a Hermitian operator using a **Cartan decomposition** and the adjoint
```

Comment:
I dont understand why, but `variational_kak_adj` doesnt render as code and the link also doesnt work 🤔

Reply:
despite the same format as the other items and link in the standard markdown notation 😕 
![image](https://github.com/user-attachments/assets/dddb7256-bf5e-48ae-980b-c8c3c2a227fb)

Reply:
Usually this can happen when there are backticks (`) next to colons (:), and sphinx thinks you're trying to specify a directive 😠

Reply:
it's not a problem in all cases above though with that same pattern 🤔

Reply:
#sphonx

Reply:
Felt very clever on this one, let's see if it works :crossed_fingers: 
![image](https://github.com/user-attachments/assets/660efbe4-0c78-41cf-b88e-c870f82c3d76)